### PR TITLE
Add more mime types to import csv validation

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | staging for features or enhancements / 3.2 for bug fixes 
| Bug fix?                               | yes/no
| New feature?                           | yes/no
| Deprecations?                          | yes/no
| BC breaks?                             | yes/no
| Automated tests included?              | yes/no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
We noticed in last days two reported csv files, which cannot pass mime types validation during make import contacts.
I've made some research and tests, I found some dicssution 
https://stackoverflow.com/questions/16234859/what-is-the-mime-types-to-upload-csv-file
In conclusion mime can return more types, If detect some leters or combination of letters.
In our cases csv return application/octet-stream and text/x-Algol68 

This is reason why I've added to validation:

- application/octet-stream
- text/*

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Try upload import file and check If import works like before

### CODE REVIEW

**Review by escopecz — APPROVED** [review]
I don't see a reason why this would hurt 👍

**Review by hluchas — APPROVED** [review]
👍 
TY!

**Comment by hluchas** [comment]
Needs codecov re-run

**Comment by RCheesley** [comment]
@hluchas do you mean that we need to rebase it to get the up to date codecov? If the files listed are out of date it usually means that the PR is out of date with Staging but I don't see that here. What is the issue?

**Comment by hluchas** [comment]
> @hluchas do you mean that we need to rebase it to get the up to date codecov? If the files listed are out of date it usually means that the PR is out of date with Staging but I don't see that here. What is the issue?

I see this:
Snímek obrazovky 2020-11-20 v 14 35 43

`codecov/patch` with red cross

**Comment by RCheesley** [comment]
Yes, it is because this patch is not covered by tests (see docs on [codecov patch checks](https://docs.codecov.io/docs/commit-status#patch-status)) and we're aiming to hit a minimum coverage that equals the project coverage - at least that's my interpretation of this. 

[Here](https://github.com/mautic/mautic/pull/9390#issuecomment-724732485) you can see it says that it does not change overall project coverage, so I think we should be good to go if the team do not feel that the changes need to be covered by automated tests? @escopecz @dennisameling any thoughts on that?

**Comment by dennisameling** [comment]
@RCheesley @escopecz IMO it's enough if project coverage remains the same. It just sets the `codecov/patch` target to `31.54%` because that's the value of `codecov/project`. The following PR allows `codecov/patch` to be `0%`, please review: https://github.com/mautic/mautic/pull/9423
